main.nf

```diff
@@ -145,7 +148,10 @@ workflow NFCORE_SAREK {
         germline_resource,
         known_indels,
         known_snps,
-        pon)
+        pon,
+        params.bbsplit_fasta_list,
+        params.bbsplit_index
```

Comment:
Given you are assigning them above, you should be able to use those here and remove the `params.` part.